Resources/doc/index.rst

```diff
@@ -1,15 +1,167 @@
+
+..
+
+   NOTE: If composer prompts input questions just press enter.
+
+
+Then run magento install script (be patient it takes some time):
+
+.. code-block:: bash
+
+    cd store
+    ./magento.sh
+
+..
+
+After Magento installation is complete, clear Magento cache:
+
+.. code-block:: bash
+
+    rm -r ./magento/var/cache/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ./magento.sh
+
+..
+
+Now You must create new index for ElasticSearch:
+
+.. code-block:: bash
+
+    cd ..
+    app/console es:index:create
+
+..
+
+And import demo content for `ongr.dev <http://ongr.dev>`_ pages:
+
+.. code-block:: bash
+
+    app/console es:index:import --raw src/ONGR/DemoBundle/Resources/data/contents.json
+
+..
+
+Now you need to import data from magento to Your newly baked ONGR shop:
```

Comment:
you seem to have lots of respect for the reader. While that is most commendable, I think it is not really necessary to capitalize "You".